services/webnn/ort/graph_builder_ort.h

```diff
@@ -94,28 +94,39 @@ class GraphBuilderOrt {
   const mojom::Operand& GetOperand(uint64_t operand_id);
   std::string GetOperandName(uint64_t operand_id);
 
-  // Some initializers must be uploaded to raw data, for example:
-  // 1. Reshape op needs parameter *shape* as raw data to do shape inference.
-  // 2. Reduce op needs parameter *axes* as raw data to do shape inference.
-  // 3. Expand op needs parameter *shape* as raw data to do shape inference.
-  // 4. Slice op needs parameter *starts*, *ends* and *steps* as raw data to do
-  // shape inference.
+  // Create a new initializer for the graph with the given shape, data and data
+  // type, return the name of the initializer.
+  //
+  // Use the raw data when:
+  // 1. The byte size of the data is less than 128.
+  // 2. The initializer is used for shape inference.
+  // Otherwise, use external data.
+  //
+  // For example, some initializers must use raw data to do shape inference, and
+  // they are almost always smaller than 128 bytes:
```

Comment:
I think 128 byte size would cover all initializers used for shape inference, because it could carry 16 x int64_t values and the existing WebNN maximum rank is 8.